src/main/java/com/ksm/wstbackoffice/domain/Warehouse.java

```diff
@@ -0,0 +1,27 @@
+package com.ksm.wstbackoffice.domain;
+
+import com.ksm.wstbackoffice.enums.PriceType;
+import com.ksm.wstbackoffice.enums.WarehouseType;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "warehouses")
+public class Warehouse {
+    @Id
```

Comment:
Please remove all hibernate related annotations.
It´s going to be the next step to add the repository, hibernate annotations, service, controller and test crud with the postman

Reply:
done